minard/templates/runselection_run.html

```diff
@@ -0,0 +1,311 @@
+{% extends "layout.html" %}
+{% block title %}Run Selection Run Detail{% endblock %}
+{% block head %}
+  {{ super() }}
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/pca.css') }}" media="screen">
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+  <!-- Start Page -->
+  <div class="container">
+
+    <!-- Page Title -->
+    <div class="row">
+      <div class="col-md-12">
+        <h1 style="text-align:center">Run Selection for Run {{ run_number }}</h1>
+      </div>
+    </div>
+
+    <!-- General Information Table -->
+    <div class="row">
+      <div class="col-md-12">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h1 class="panel-title"><b>General information</b></h1>
+          </div>
+          <div class="panel-body">
+            <div class="row">
+              <div class="col-md-12">
+                <table class="table table-striped">
+                  <thead>
+                    <tr>
+                      <th width="50.0%">Parameter</th>
+                      <th width="50.0%">Value</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {% for parameter in general_info.keys() %}
+                    <tr>
+                      <td>{{ parameter }}</td>
+                      {% if general_info[parameter] == "Pass" %}
+                        <td><font color="green">Pass</font></td>
+                      {% elif general_info[parameter] == "Fail" %}
+                        <td><font color="red">Fail</font></td>
+                      {% else %}
+                        <td>{{ general_info[parameter] }}</td>
+                      {% endif %}
+                    </tr>
+                    {% endfor %}
+                  </tbody>
+                </table>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Run Selection Results Title -->
+    <div class="row">
+      <div class="col-md-12">
+        <h1 style="text-align:left">Run Selection Results</h1>
+      </div>
+    </div>
+
+    <!-- Run Selection Results Collapsable Tabs -->
+    {% for criteria in display_info.keys() %}
+      <div class="row">
+        <div class="col-md-12">
+          <div class="panel-group">
+            <div class="panel panel-default">
+    
+              <!-- Collapsable header title for criteria tag -->
+              <div class="panel-heading">
+                <h4 id="title-{{criteria}}" class="panel-title">
+                  <a data-toggle="collapse" href="#collapse{{criteria}}">
+                    {% if display_info[criteria]["criteria_result"] == true %}
+                      <b><font color="green">{{ criteria }}</font></b>
+                    {% elif display_info[criteria]["criteria_result"] == false %}
+                      <b><font color="red">{{ criteria }}</font></b>
```

Comment:
This is fairly minor, but I find that the result is "failed" for the "run_list_opinion" when the run has not yet been evaluated by the shifter. It might be worth thinking about having a "pending" color, if possible.